lib/backend/k8s/conversion/conversion_test.go

```diff
@@ -1789,46 +1817,204 @@ var _ = Describe("Test Namespace conversion", func() {
 
 	It("should handle NetworkPolicy resource versions", func() {
 		By("converting crd and k8s versions to the correct combined version")
-		rev := c.JoinNetworkPolicyRevisions("1234", "5678")
+		rev := c.JoinRevisions("1234", "5678")
 		Expect(rev).To(Equal("1234/5678"))
 
-		rev = c.JoinNetworkPolicyRevisions("", "5678")
+		rev = c.JoinRevisions("", "5678")
 		Expect(rev).To(Equal("/5678"))
 
-		rev = c.JoinNetworkPolicyRevisions("1234", "")
+		rev = c.JoinRevisions("1234", "")
 		Expect(rev).To(Equal("1234/"))
 
 		By("extracting crd and k8s versions from the combined version")
-		crdRev, k8sRev, err := c.SplitNetworkPolicyRevision("")
+		crdRev, k8sRev, err := c.SplitRevision("")
 		Expect(err).NotTo(HaveOccurred())
 		Expect(crdRev).To(Equal(""))
 		Expect(k8sRev).To(Equal(""))
 
-		crdRev, k8sRev, err = c.SplitNetworkPolicyRevision("/")
+		crdRev, k8sRev, err = c.SplitRevision("/")
 		Expect(err).NotTo(HaveOccurred())
 		Expect(crdRev).To(Equal(""))
 		Expect(k8sRev).To(Equal(""))
 
-		crdRev, k8sRev, err = c.SplitNetworkPolicyRevision("1234/5678")
+		crdRev, k8sRev, err = c.SplitRevision("1234/5678")
 		Expect(err).NotTo(HaveOccurred())
 		Expect(crdRev).To(Equal("1234"))
 		Expect(k8sRev).To(Equal("5678"))
 
-		crdRev, k8sRev, err = c.SplitNetworkPolicyRevision("/5678")
+		crdRev, k8sRev, err = c.SplitRevision("/5678")
 		Expect(err).NotTo(HaveOccurred())
 		Expect(crdRev).To(Equal(""))
 		Expect(k8sRev).To(Equal("5678"))
 
-		crdRev, k8sRev, err = c.SplitNetworkPolicyRevision("1234/")
+		crdRev, k8sRev, err = c.SplitRevision("1234/")
 		Expect(err).NotTo(HaveOccurred())
 		Expect(crdRev).To(Equal("1234"))
 		Expect(k8sRev).To(Equal(""))
 
 		By("failing to convert an invalid combined version")
-		_, _, err = c.SplitNetworkPolicyRevision("1234")
+		_, _, err = c.SplitRevision("1234")
 		Expect(err).To(HaveOccurred())
 
-		_, _, err = c.SplitNetworkPolicyRevision("1234/5678/1313")
+		_, _, err = c.SplitRevision("1234/5678/1313")
 		Expect(err).To(HaveOccurred())
 	})
 })
+
+var _ = Describe("Test ServiceAccount conversion", func() {
+
+	// Use a single instance of the Converter for these tests.
+	c := Converter{AlphaSA: true}
+
+	It("should parse a ServiceAccount to a Profile", func() {
```

Comment:
I think this is actually testing the "default" namespace case.

If so, this test case is a bit contrived, because in a real system the objects returned will always have the ObjectMeta.Namespace field populated (even for "default")

Reply:
i.e. it's not any different from the "in a namespace" test below.